site/_config.yml

```diff
@@ -52,6 +52,18 @@ defaults:
       version: master
       gh: https://github.com/vmware-tanzu/sonobuoy/tree/master
       layout: "docs"
+  - scope:
```

Comment:
Ah, this is an issue with the docs generation script. If you run it twice, it will just add the same entry. You will need to remove the duplicate values here and in the `versions` list below.